image_metadata_script.py

```diff
@@ -0,0 +1,53 @@
+import os, time
+import filetype
+import csv
+import sys
+
+minio_link = 'None'
+dataset = 'None'
+is_processed = 0
+
+
+def image_metadata(path):
```

Comment:
when you call this function from another python file, will the variables `minio_link`, `dataset`, `is_processed` still there?

Reply:
It worked when I called the function from another python file which I understand is an unusual behaviour since the variables are initialised outside the function. However, to avoid any further issues, I have moved those 3 statements within the function body.